cgi-bin/LJ/S2Theme/headsup.pm

```diff
@@ -17,9 +17,10 @@ sub header_props {
     my $self = shift;
     my @props = qw(
         image_foreground_header_url
-        int image_foreground_header_height
-        string image_foreground_header_alignment
+        image_foreground_header_height
```

Comment:
I should have explained, sorry. We don't mention variable type (int, string, whatever) when sorting new properties. This is just clean up and making sure this file is consistent with other similar files.
And now I've just noticed I left the 'string' for image_foreground_header_alignment. :(